frameworks/g-cloud-12/questions/services/standardsISOIEC27001Who.yml

```diff
@@ -13,8 +13,8 @@ max_length_in_words: 10
 
 validations:
   - name: answer_required
-    message: You need to answer this question.
+    message: Enter the accreditor name.
   - name: under_10_words
-    message: Your description must be no more than 10 words.
+    message: Accreditor name must be 10 words or fewer.
   - name: under_character_limit
```

Comment:
Query: are there 2 separate limitations for this field - word and character count? Could we limit it to word count only?